python/idsse_common/test/test_netcdf_io.py

```diff
@@ -65,9 +68,9 @@ def test_read_netcdf(example_netcdf_data: Tuple[Dict[str, any], ndarray]):
     assert grid.shape == (1597, 2345)
     x_dimensions, y_dimensions = grid.shape
 
-    assert grid[0][0] == approx(72.98599)
-    assert grid[round(x_dimensions / 2)][round(y_dimensions / 2)] == approx(12.505991)
-    assert grid[x_dimensions - 1][y_dimensions - 1] == approx(2.4259913)
+    assert grid[0][0] == approx(72.80599)
+    assert grid[round(x_dimensions / 2)][round(y_dimensions / 2)] == approx(26.005991)
+    assert grid[x_dimensions - 1][y_dimensions - 1] == approx(15.925991)
```

Comment:
I ran those tests again in DEBUG, and those are indeed the values in the grid at those pixel coords ((798, 1172) and (1597, 2345)).

Reply:
I've been trying to logically verify the values...
Are those believable temperatures, in early November 2022, for central Nebraska (26.0 F), and Newfoundland, Canada (15.9 F)? Converting the pixels to geo coordinates, that's where those points appear on the map.

I can't validate the true forecast value with tools like [NBM 1D Viewer](https://hwp-viz.gsd.esrl.noaa.gov/wave1d/?location=KGUY&selectedgroup=Default&col=1&hgt=1&obs=true&fontsize=1.2&darkmode=on&graph=fa-chart-bar&tz=UTC) because the Points in the unit test are random, and don't match any of the obs stations expected by 1D Viewer.